src/full/Agda/Main.hs

```diff
@@ -112,6 +112,7 @@ runAgdaWithOptions backends generateHTML interaction progName opts
             printStatistics 1 Nothing =<< useTC lensAccumStatistics
   where
     checkFile = Just <$> do
+      opts <- addTrustedExecutables opts
```

Comment:
A corrupt executables config file should not stop Agda from working unless the exec features are use, for simplicity say unless the `--allow-exec` option is supplied.

Is this guaranteed or should `addTrustedExecutables` be relativized to presence of this option?

Reply:
The problem is that the `executables` file is parsed once when Agda starts, and `--allow-exec` is a pragma option. A solution might be to issue a warning for missing or non-executable entries and drop them from the list, rather than hard-failing.

Reply:
Yes, that'll do!

Reply:
I'll change the error to a warning.

Reply:
This lead me down a rabbit hole, where library warnings have tons of infrastructure around them, which library errors do not. Should be fixed now, will push after testing.